src/core/README.md

```diff
@@ -0,0 +1,83 @@
+# Settlement Core
+
+> ### Framework for building Interledger settlement engines in JavaScript
+
+## Design Goals
+
+- **Isomorphic**. JavaScript settlement engines should operate seamlessly across Node.js, desktop & mobile browsers, and Electron apps.
+- **Interoperable**. Settlement engines should fully support each next-generation connector, including [Interledger.rs](https://github.com/interledger-rs/interledger-rs), [Rafiki](https://github.com/interledgerjs/rafiki), and the [Java connector](https://github.com/sappenin/java-ilpv4-connector/).
+- **Accessible**. Provide the essential primitives to quickly develop safe, reliable settlement engine implementations.
+- **Modular**. TODO
+- **Scalable**. Support standalone clients all the way up to high-volume, low-latency service providers.
+
+## Prerequisites
+
+Before building a settlement engine, [please read the Settlement Engine RFC](https://github.com/interledger/rfcs/blob/76c717604ee8d51d8f61a9bc2cb92ba135738f09/0000-settlement-engines/0000-settlement-engines.md).
+
+## API
+
+### Building a settlement engine
+
+TODO Add some docs here to explain how to do this
+
+TODO Update RFC link after publish
+
+TODO 🚨that this is beta and subject to change
+
+Settlement engines have to primary responsibilities:
+
+1. Defining a `settle` function that given some amount, asynchronously performs a settlement, and returns the amount that was sent.
+2. Listening for incoming settlements, and invoking a `creditSettlement` callback with the amount received.
+
+All interaction with the connector, accounting, and balances are handled by `settlement-core`.
+
+TODO Include annotated settlement engine interface
+
+TODO Include annotated account services
+
+TODO Explain more about what the core lib does and doesn't do
+
+```js
+export const connectEngine = async ({ sendMessage, creditSettlement }) => {
```

Comment:
Have to laugh a bit because this is so similar to the Ledger Plugin Interface V2 with `accountId`s added